src/Config.php

```diff
@@ -154,7 +154,7 @@ public function getUsingCache(): bool
         return $this->usingCache;
     }
 
-    public function registerCustomFixers(iterable $fixers): ConfigInterface
+    public function registerCustomFixers(iterable $fixers): self
```

Comment:
@kubawerlos , `: static` + `@return $this` ?

Reply:
@keradus [not possible](https://3v4l.org/SNQBj#v7.4.33), we need to drop support for PHP 7.4 as soon as possible and I believe it doesn't even require v4, as the public API won't be changed. Laminas packages for example bump minimal PHP version in minor versions (example [here](https://github.com/laminas/laminas-code/releases/tag/4.8.0), [here](https://github.com/laminas/laminas-diactoros/releases/tag/2.18.0) and [here](https://github.com/laminas/laminas-httphandlerrunner/releases/tag/2.3.0)). If we drop 7.4 so many things are going to be unlocked for development 🥹.

<img width="1048" alt="image" src="https://github.com/user-attachments/assets/0529c4ee-4d6f-4ae6-aeea-e3d5ffcbc686" />

Reply:
I agree that bumping minimal php version is not a bc break (symfony sometimes does this in minor updates as well).
But changing `self` to `static` (native return type) would be a bc break (for non-final classes).